### ISSUE TITLE
Death is not punishing enough

### ISSUE THREAD

**Issue by Guineapigboyx** [uthw/Mincemeat-2#6]
_I suck at this game and I'm saying this._

When I'm playing it just feels like death is just not a concern when playing since I will always know I can get back without losing anything other than the time and the flow of doing something. I even sometimes feel like it is better to die, like I forgot to grab torches but I'm already deep in a ravine so it came by me why don't I just kill myself and just grab torches on the surface and then go back down?

I think a system like minced one worked pretty well _(almost)_ where teleporting back was a thing you could only do if you had specific materials. or just really suck and die a lot.
the only thing is maybe don't ender pearls since finding multiple ender pearls/endermen can be a pain at times and is very rare to find excluding quests.
I would rather have a system where it costs an item than something like exp because I feel exp can be hard annoying to grind for after you die and lose it all but it can be fun to grind for an item and does not punish you as hard since you could always walk back. 
I think you should also be able to run out of teleports back if you die too many times. for example, you would not have any ender pearls and now have to walk.

Real experience to build off of:
_I remember one time I died in Ever Dawn and I had to find out how I could back to my stuff which at this point my stuff was too far away to walk to, thanks to my trusty grappling hook. 
But I just ran out of Ender pearls and i did not get an enchanted grave key and my spawn point was at a binding dungeon so what I did was I walked back to the portal and tried to find random structures in the hope I could find some Ender pearls what I did after a little over 2 hours of hunting I got enough to make an enchanted grave key and teleported back. this hole experience made me get to experience more of the mod pack and also go on a side quest to get my stuff back which I found fun._

tldr:
I want a system that punishes you for death but does not force you to walk and instead makes something else you can do to get back on your feet.
